src/views/design/ScriptDetail/DetailActions.tsx

```diff
@@ -99,33 +100,41 @@ function DetailActions({
     return (
         <Box display="flex" alignItems="center" justifyContent="space-between" marginX={2.2}>
             <Box flex="0 0 auto">
-                <Tooltip
-                    title={translator('scripts.detail.main.actions.add_action')}
-                    enterDelay={1000}
-                    enterNextDelay={1000}
-                >
-                    <IconButton
-                        aria-label={translator('scripts.detail.main.actions.add_action')}
-                        className={classes.addButton}
-                        onClick={onAdd}
-                        color="default"
-                    >
-                        <AddIcon />
-                    </IconButton>
-                </Tooltip>
+                {checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC')
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'